aviasales/src/app/components/button/button.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-button',
+  templateUrl: './button.component.html',
+  styleUrls: ['./button.component.sass']
+})
+export class ButtonComponent implements OnInit {
+
+  constructor() { }
```

Comment:
Delete unneeded hooks and constructors, if eslint has error, set settings for these rules